atomic_reactor/plugins/pre_add_filesystem.py

```diff
@@ -180,6 +202,7 @@ def parse_image_build_config(self, config_file_name):
 
         config = ConfigParser()
         config.readfp(self.get_default_image_build_conf())
+        self.update_config_from_dockerfile(config)
```

Comment:
Can we call this method from within `get_default_image_build_conf` ?

Reply:
Sounds better :+1:,

it can also encapsulate 
```
config = ConfigParser()
config.readfp(self.get_default_image_build_conf())
... default stuff from dockerfile ....
```

Reply:
updated